netlify.toml

```diff
@@ -4,7 +4,7 @@
     publish = "docs/book/book"
 
 [build.environment]
-    GO_VERSION = "1.21"
+    GO_VERSION = "1.21.6"
```

Comment:
```suggestion
    GO_VERSION = "1.21.7"
```

Reply:
Thanks for the quick review. Just fixed the patch version.